### PR TITLE
add info about DID and idena

### PR DESCRIPTION
**Pull request by itoukaijiE**

### CODE REVIEW

**Review by jhfnetboy — COMMENTED** [review]
Do you have reference links ? 
or  Do Idena has any graph flow to show the main life cycle? It will help us more^_^

File: dao/DID/idena.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
+# All You Need To Know About Idena
+
+## What is Idena
+
+Tl;NR
+
+Idena is a technique to prove YOU are a HUMAN without KYC.
+
+**maybe the most important usage for Idena currently is the gitcoin trust, it worth 50% trust.**
```
[the remark this answers is not in the document]
Comment (jhfnetboy) [reply dao/DID/idena.md:9]: Mac app is not very familiar to use...just share the experiments

[the remark this answers is not in the document]
Comment (itoukaijiE) [reply dao/DID/idena.md:9]: add reference links.
do not want make this introduction too long to read, more info is leave for the users to dig into.
also, idena app is https://app.idena.io/, just like metamask.

**Review by liberhe — DISMISSED** [review]

**Review by liberhe — APPROVED** [review]

**Review by xrdavies — APPROVED** [review]
Looks good to me.

**Review by jhfnetboy — APPROVED** [review]

**Pull request merged**